other/rec-req/require-base-image/99-delete.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: kuttl.dev/v1beta1
-kind: TestStep
-delete:
-- apiVersion: kyverno.io/v1
-  kind: ClusterPolicy
-  name: require-base-image
-commands:
-- command: kubectl delete all --all --force --grace-period=0 -n require-base-image-ns
```

Comment:
Is the force deletion still required?

Reply:
It cased a timeout issue [here](https://github.com/kyverno/policies/actions/runs/6170604599/job/16747199526?pr=750) which led to a flaky test.

Reply:
It's still needed in some very specific cases, but looking through this test case I don't think it's needed in this one.